internal/form/repo.go

```diff
@@ -69,6 +70,11 @@ func (f MigrateRepo) ParseRemoteAddr(user *db.User) (string, error) {
 		if err != nil {
 			return "", db.ErrInvalidCloneAddr{IsURLError: true}
 		}
+
+		if netutil.IsLocalHostname(u.Hostname()) {
+			return "", db.ErrInvalidCloneAddr{IsURLError: true}
+		}
```

Comment:
@michaellrowley This is where the real fix to the report as far as I understand.

Reply:
Yeah, that seems to be the case (although without the rewritten ``IsLocalHostname`` it would still have been possible to bypass this call and the one in ``validateWebhook``)